v4/source/plugins.yaml

```
@@ -102,6 +134,17 @@
           $ref: '#/responses/Forbidden'
         '501':
           $ref: '#/responses/NotImplemented'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/platform/model"
+
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            pluginID := "4xp9fdt77pncbef59f4k1qe83o"
+
+            ok, resp = Client.RemovePlugin(pluginID)
```

Plugin ID's are actually more meaningful the other id's in Mattermost. Let's use a real world example here:
```suggestion
            ok, resp = Client.RemovePlugin("com.mattermost.demo-plugin")
```

---

done